### ISSUE TITLE
Show example output of functions in docs

### ISSUE THREAD

**Issue by theryansmee** [ngneat/falso#69]
### Description

It would be good to have an example output for each function example

```
/**
 * Generate a random rgb.
 *
 * @category colors
 *
 * @example
 *
 * randRgb()
 * // rgb(69, 254, 28)
 *
 * @example
 *
 * randRgb({ alpha: true })
 * // rgba(159, 167, 129, 0.74)
 *
 * @example
 *
 * randRgb({ length: 3 })
 * // [ 'rgb(158, 13, 22)', 'rgb(58, 214, 87)', 'rgb(43, 193, 36)' ]
 * 
 */

```

### Proposed solution

I'm not sure if these will be pulled from the JS Doc comments automatically, or if we will need to do some extra work to our docusaurus config

### Do you want to create a pull request?

No

**Comment by shaharkazaz** [comment]
@theryansmee I'm adding an interactive playground see #62, So I think people can try it out there.
@NetanelBasal WDYT?

**Comment by NetanelBasal** [comment]
Yes, it should be resolved in #62. Thanks.
